aggregator/aggregator.go

```diff
@@ -393,7 +395,7 @@ func (agg *Aggregator) sendNewCheckpointTask() {
 		fromTimestamp = 0
 	}
 
-	toTimestamp := block.Time()
+	toTimestamp := block.Time() - uint64(types.MESSAGE_SUBMISSION_TIMEOUT.Seconds())
```

Comment:
This should also include the aggregation timeout (see #212). It makes sense because worst-case is aggregation + submission timeout.

Reply:
Adjusted to include `- uint64(types.MESSAGE_BLS_AGGREGATION_TIMEOUT.Seconds())`